lib/registry.js

```diff
@@ -201,12 +201,24 @@ registry.configureModel = function(ModelCtor, config) {
       'must be an array of objects', modelName);
   }
 
+  // Remote methods
+  if (typeof settings.methods === 'object' && settings.methods !== null) {
+    var methods = settings.methods = settings.methods || {};
+    Object.keys(settings.methods).forEach(function(key) {
+      ModelCtor.remoteMethod(key, settings.methods[key]);
```

Comment:
Please rework this part according to https://github.com/strongloop/loopback/issues/209#issuecomment-77103970, so that:

 - key `prototype.methodName` is mapped to `remoteMethod('methodName', { isStatic: false, ... })`
 - key `methodName` is mapped to `remoteMethod('methodName', { isStatic: true, ... })`